golem/payload/__init__.py

```diff
@@ -9,7 +9,7 @@
     PropertyValue,
 )
 from golem.payload.exceptions import InvalidProperties, PayloadException
-from golem.payload.parsers import PayloadSyntaxParser, SyntaxException
+from golem.payload.parsers import PayloadSyntaxParser, SyntaxException, TextXPayloadSyntaxParser
```

Comment:
`TextXPayloadSyntaxParser` is a specific case of, in general, optional implementation, hence it was not in a typical import tree, but in a totally separate path which requires from the user a full deep import path. Looks like this needs to be reverted.